### ISSUE TITLE
differing output lintr-bot vs local lint_package

### ISSUE THREAD

**Issue by AdamSpannbauer** [jimhester/lintr#378]
We added lintr-bot to a project, and it takes issue with a lot more of the code than when `lintr::lint_package()` is run locally.

In particular, we receive *a lot* of warnings around global functions/variables (similar to what's documented in #27).  These warnings are not reported when running locally.

What is different about how the 2 processes are run and is there a way that we can get lintr-bot on the same page as the local check?

----

FYI:
```r
> packageVersion('lintr')
[1] ‘1.0.3’
```

**Comment by AdamSpannbauer** [comment]
Just saw this in the README.md.... closing issue

> ```
> [after_success:
>   - R CMD INSTALL $PKG_TARBALL
>   - Rscript -e 'lintr::lint_package()'](url)
> ```

**Comment by dpprdan** [comment]
@AdamSpannbauer I made the same mistake and this issue helped to track that down. So thanks for putting this out here!

**Comment by janclod** [comment]
Thank you for posting this. Also helped me to figure out why my GHA build broke, while my local build did not ([issue](https://github.com/janclod/testPipes)).
